### DISCUSSION TITLE
特定のキーでピンセットによるショート入力は可能だがキースイッチ入力では反応しない（ダイオードとキーソケットを直接つなぐ？）

### DISCUSSION THREAD

**Discussion by mizuki0101** [Yowkees/keyball#647] [Q&A] [answered]
[現状の問題点]
3と4のキーがピンセットでのショートでは入力ができますば、キースイッチを差し込んで入力を行うと反応しません。
ハンダが原因かと思い何度かやり直しましたが改善されず、
付け直しの際ににキーソケットに基盤のランドが剥がれてしまいました。。。
・ピンセットでのショートでの入力可能
・キースイッチを付けての入力が不可
・キーソケットのランドが剝がれてしまっている。（剝がれた後もショートでの入力は可能）

<img width="1171" alt="1png" src="https://github.com/user-attachments/assets/48316abd-368f-4c3f-abfb-f4e4ea0d6c30">
<img width="1304" alt="2" src="https://github.com/user-attachments/assets/446029d2-8a1a-4bda-ab30-b949f83562d2">
<img width="1079" alt="5" src="https://github.com/user-attachments/assets/8fe520c1-effb-44b2-915c-9b4885244672">
<img width="1304" alt="3" src="https://github.com/user-attachments/assets/8131afd7-846e-46e2-b58d-358bcdcbca1e">

こちらの質問からダイオードとキーソケットを直接繋げば改善される書き込みがあったため検証を進めていますが、
https://github.com/Yowkees/keyball/discussions/412
対応しているダイオードとキーソケットの場所が分からないためご教授いただけないでしょうか？
（近いダイオードに対応しているかと思いましたが、そうではないようで基盤から確認することができませんでした。。。）
3と4両方ともランドが剝がれているため、両方の対応箇所を確認できればと思います。

また、ダイオードとキーソケットをつなぐ時には添付画像の銅線を使用予定ですが問題ないでしょうか？
<img width="1304" alt="4" src="https://github.com/user-attachments/assets/5ef70ba0-1797-48ad-bdeb-c8c89ab2c4df">

上記の質問については私なりに検証している内容であり、
本事象の改善方法が他にあれば合わせて教えていただけると助かります。

よろしくお願いいたします。

**Comment by mizuki0101** [comment, marked as answer]
こちら自己解決しました！
下記ファイルにPCBの配置図があったため、キーソケットとダイオードの対応する箇所を確認できました。
keyball39/design_data/keyball39_Left_KiCad.zip
とてもコンパクトでこれから愛用していけそうです。ありがとうございます。

![IMG_2478](https://github.com/user-attachments/assets/3a61a76b-02a6-4dc5-9933-4e8252ae1848)

**Comment by Yowkees** [comment]
夏季休業でご連絡が遅くなり申し訳ありません。
ジャンパー線の追加でご自身で解決いただき、誠にありがとうございました。
ぜひ末永くご愛用いただければ幸いです！
